README.md

```diff
@@ -1 +1,17 @@
-# angular-scope-watch-lab
\ No newline at end of file
+# Watching Model changes
+
+## Objectives
+
+- Integrate $scope.$watch to watch Controller properties
+- Use a deep $watch to observe deep changes
+- Write a $watchCollection method for better shallow performance
+
+## Instructions
+
+In this repo we've got a few fields to fill out for a user's information.
```

Comment:
list out what the different fields are to help the reader get familiar with the codebase. Give a general overview of what the domain is.